### PR TITLE
feat(core): add `app-region: drag` to `data-tauri-drag-region`

### PR DESCRIPTION
**Pull request by Legend-Master**
Use `app-region: drag` on supported platforms (currently only WebView2 123+ on Windows)

### CODE REVIEW

File: core/tauri/src/window/scripts/drag.js [hunk 1]
```diff
@@ -63,4 +63,9 @@
       }
     })
   }
+  if ('adoptedStyleSheets' in document) {
```
Comment (amrbashir) [inline core/tauri/src/window/scripts/drag.js]:
```suggestion
  if (osName === 'windows' && 'adoptedStyleSheets' in document) {
```

**Review by amrbashir — CHANGES_REQUESTED** [review]
We probably need to get the webview version and embed it into this script at startup then disable the custom JS handler on Windows when using WebView2 123+

**Comment by Legend-Master** [comment]
> We probably need to get the webview version and embed it into this script at startup then disable the custom JS handler on Windows when using WebView2 123+

I don't think we need to, we won't get events in non client region, so if `app-region: drag` is working, it should just skip the other handlers

File: core/tauri/src/window/scripts/drag.js (hunk 1 above)
Comment (Legend-Master) [reply core/tauri/src/window/scripts/drag.js]: We might get `app-region: drag` in the future on Linux and Mac, do we want to limit this part to only run on Windows?

Comment (amrbashir) [reply core/tauri/src/window/scripts/drag.js]: yes, until other platforms support, we should limit to only Windows

**Comment by amrbashir** [comment]
> > We probably need to get the webview version and embed it into this script at startup then disable the custom JS handler on Windows when using WebView2 123+
> 
> I don't think we need to, we won't get events in non client region, so if `app-region: drag` is working, it should just skip the other handlers

That's enough I guess

File: .changes/app-region-drag.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"tauri": patch:feat
+---
+
+Add `app-region: drag` to HTML elements with `data-tauri-drag-region` on supported platforms (currently only WebView2 123+ on Windows)
```
Comment (amrbashir) [inline .changes/app-region-drag.md]:
```suggestion
Add `app-region: drag` to HTML elements with `data-tauri-drag-region` on Windows, only WebView2 123+, which should fix dragging using touch.
```

**Review by amrbashir — CHANGES_REQUESTED** [review]

**Review by amrbashir — APPROVED** [review]

**Pull request merged**